src/admin-debug-info.php

```diff
@@ -79,6 +75,6 @@ public function handle_submit() {
 			$this->option->set( 'show_options_debug', isset( $_POST['show_options_debug'] ) );
 		}
 
-		wp_safe_redirect( self_admin_url( 'tools.php?page=' . apply_filters( 'yoast_version_control_admin_page', '' ) ) );
+		wp_safe_redirect( self_admin_url( 'tools.php?page=' . apply_filters( 'Yoast\WP\Test_Helper\admin_page', '' ) ) );
```

Comment:
Is this test helper available publicly ? If so, the old filter name should be deprecated.
See: https://github.com/Yoast/Wiki/wiki/How-to-rename-a-hook

:point_up_2: This question applies to all renamed hooks in the test helper.

Reply:
It wasn't before this.